index.js

```diff
@@ -6,6 +6,13 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
     const defaultBranch = payload.ref === 'refs/heads/' + payload.repository.default_branch
 
     const config = await context.config('settings.yml', {}, { arrayMerge: mergeArrayByName })
+    const checkOptions = {
+      owner: context.repo().owner,
+      repo: context.repo().repo,
+      name: "Settings Probot",
```

Comment:
might be worth using the name of the app, as it appears on the configuration screen

```suggestion
      name: "Settings",
```